bigquery/google/cloud/bigquery/job.py

```diff
@@ -1999,6 +2001,13 @@ def default_dataset(self):
         to use for unqualified table names in the query or :data:`None` if not
         set.
 
+        The ``dataset`` setter accepts:
```

Comment:
Is this the conventional way to document this?

Reply:
I guess? The closest I found is http://www.sphinx-doc.org/en/1.5/ext/example_google.html

```
    @property
    def readwrite_property(self):
        """list(str): Properties with both a getter and setter
        should only be documented in their getter method.

        If the setter method contains notable behavior, it should be
        mentioned here.
        """
        return ['readwrite_property']

    @readwrite_property.setter
    def readwrite_property(self, value):
        value
```

I guess I could try a definition list instead of a bulleted list if you'd prefer?

Reply:
I think in the past, we've usually just used the normal `Args:` to documment the setter args in the getter docstring

Reply:
It seems a bit odd to have "parameters" for a property setter.

![image](https://user-images.githubusercontent.com/247555/55098330-d0c03380-507a-11e9-9b2b-2e3739353fab.png)

I think I prefer the bulleted list.